examples/remote_procedure/mnist/run_task.py

```diff
@@ -104,14 +59,13 @@ def handle_with_model(
 
     shared_state = SharedRemoteProcedureTaskState(
         static_task_data=tasks,
-        validate_onboarding=onboarding_always_valid,
         function_registry=function_registry,
     )
 
+    task_dir = cfg.task_dir
     build_and_return_custom_bundle(task_dir)
-    operator = Operator(db)
 
-    operator.validate_and_run_config(cfg.mephisto, shared_state)
+    operator.launch_task_run(cfg.mephisto, shared_state)
     operator.wait_for_runs_then_shutdown(skip_input=True, log_rate=30)
 
 
```

Comment:
Wow, what a difference in this file. Really like how the essential complexity of the task takes the stage (mnist-related stuff), and the accidental complexity is minimized (library setup)